test-app/tests/integration/components/toucan-form/form-textarea-test.gts

```diff
@@ -31,4 +31,71 @@ module('Integration | Component | ToucanForm | Textarea', function (hooks) {
 
     assert.dom('[data-textarea]').hasAttribute('readonly');
   });
+
+  test('it renders `@label` and `@hint` component arguments', async function (assert) {
+    const data: TestData = {
+      text: 'multi-line text',
+    };
+
+    await render(<template>
+      <ToucanForm @data={{data}} as |form|>
+        <form.Textarea @label="Label" @hint="Hint" @name="text" />
+      </ToucanForm>
+    </template>);
+
+    assert.dom('[data-label]').exists();
+    assert.dom('[data-hint]').exists();
+  });
+
+  test('it renders a `:label` named block', async function (assert) {
+    const data: TestData = {
+      text: 'multi-line text',
+    };
+
+    await render(<template>
+      <ToucanForm @data={{data}} as |form|>
+        <form.Textarea @name="text">
+          <:label><span data-label-block>Label</span></:label>
+        </form.Textarea>
+      </ToucanForm>
+    </template>);
+
+    assert.dom('[data-label-block]').exists();
+  });
+
+  test('it renders a `:hint` named block', async function (assert) {
```

Comment:
Suggestion:
```suggestion
  test('it renders a `:hint` named block with a @label arg', async function (assert) {
```

Reply:
Should there be a test for a `:label` block with a `@hint` arg?

Reply:
Yeah let me get that added!

Reply:
Went ahead and added via https://github.com/CrowdStrike/ember-toucan-core/pull/149/commits/0ac02499eacbe66df93846a5d4bd5c950a7fd9c2